pom.xml

```diff
@@ -27,17 +27,51 @@
       <version>4.0.1</version>
     </dependency>
 
+    <!--Date/Time-->
+    <dependency>
+    <groupId>joda-time</groupId>
+    <artifactId>joda-time</artifactId>
+    <version>2.10</version>
+</dependency>
```

Comment:
It was to help with the date conversion @josh-hdz ! I was doing quite a bit of research since I kept getting errors about converting Strings to Longs and formatting and a lot of code regarding on how to convert a date string to a long had Joda-Time. Joda-Time had better Date features than Java calendar but I didn't know Java.time now has that API included so I forgot to delete it. I was messing around with how exactly I was going to convert the date and time to epoch time (which I'm still probably going to edit the method since I was overthinking the whole thing) but I ended up just parsing the string using SimpleDateFormat and the the getTime() which returns the milliseconds.